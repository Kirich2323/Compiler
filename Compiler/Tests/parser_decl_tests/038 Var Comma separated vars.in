var
    a, b: integer;
    c, d, e, g, f, k, l, m, v, x, z : float;