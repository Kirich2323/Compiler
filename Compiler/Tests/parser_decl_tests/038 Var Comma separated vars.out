main
----------
char                     type 
float                    type 
integer                  type 
a                         var         integer
b                         var         integer
c                         var           float
d                         var           float
e                         var           float
g                         var           float
f                         var           float
k                         var           float
l                         var           float
m                         var           float
v                         var           float
x                         var           float
z                         var           float